Automattic/jetpack
# Add client identification to Boost Speed Score

## Proposed changes:

* This PR adds a new $client argument to the `Speed_Score` class. This new argument is required, so new additions do not miss it when instancing the class.
* The PR also adds the client string to the 2 places where the class is being called

### Other information:


## Jetpack product discussion


N/a

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Checkout this branch locally
* Run `jetpack build plugins/jetpack && jetpack install plugins/jetpack`
* Update `projects/packages/boost-speed-score/src/lib/class-wpcom-boost-api-client.php`
* Right after line 26 (where the method `post` starts), add `l( $payload );`
* Run `jetpack docker tail` to check the requests made from your site
* Search for `/jetpack-boost/speed-scores/` and make sure the payload includes `client` with the right value (`boost-plugin` if you are fetching the scores from the Boost plugin, and `jetpack-dashboard` if it's from the Jetpack Dashboard)
